### ISSUE TITLE
U+2064 to U+206f are 0 width, but wcwidth returns 1 width

### ISSUE THREAD

**Issue by bspammer** [jquast/wcwidth#26]
```python3
base = ord('\u2064')
for i in range(12):
    print('A' + chr(base+i) + 'B')
```

**Comment by jquast** [comment]
In iTerm2, these are all displayed as an empty space.
What terminal/s are you using that displays these as 0 wide? I will try some others given time

**Comment by jquast** [comment]
From http://www.unicode.org/Public/13.0.0/ucd/extracted/DerivedGeneralCategory.txt
```
2060..2064    ; Cf #   [5] WORD JOINER..INVISIBLE PLUS
```

**Comment by tiptenbrink** [comment]
This is still an issue, see the linked issue above.

**Comment by jquast** [comment]
I will do my best to address this in next release, this is the FOSS project I most wish to volunteer my time into, I just pushed my WIP branch https://github.com/jquast/wcwidth/compare/use-jinja?expand=1 as I hope I can then better generate code for other languages using jinja and get these issues fixed more universally.

But anyway I start a new job today, so that's life!

**Comment by jquast** [comment]
This is closed in today's release by #91
